README.md

```diff
@@ -16,11 +16,9 @@ Add the plugin to your Webpack configuration:
 
   resolve: {
     plugins: [new BowerResolvePlugin()],
-    modules: [
-        'bower_components'
-    ],
-    descriptionFiles: ['bower.json', 'package.json'],
-    mainFields: ['main', 'browser']
+    modules: ['bower_components', 'node_modules'],
+    descriptionFiles: ['.bower.json', 'package.json'],
```

Comment:
Yep, I don't really like the way webpack handles it now as well. I never wrote any webpack plugin or used `enhance-resolve`, but I can have a look. Can you keep the PR open in the meantime? It may help others with the same problem.